c++/src/kj/compat/http.c++

```diff
@@ -4312,6 +4342,12 @@ public:
   uint64_t sentByteCount() override { return out->sentByteCount(); }
   uint64_t receivedByteCount() override { return in->sentByteCount(); }
 
+  kj::Maybe<kj::String> getPreferredExtensions(ExtensionsContext ctx) override {
+    // TODO(now): Do we want in or out?
```

Comment:
Thanks for writing this, it was very helpful. Also, sorry for not getting back to this until now. I had briefly skimmed over it during holidays and figured this would be a lot more work, so when other stuff came up I avoided the context switch... Should've just read this properly :man_facepalming: